google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/LinkManagerToClient.java

```diff
@@ -0,0 +1,164 @@
+import com.google.ads.googleads.v1.utils.ResourceNames;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * This example demonstrates how to link an existing Google Ads manager customer to an existing
+ * Google Ads client customer.
+ */
+public class LinkManagerToClient {
+
+  private static class LinkManagerToClientParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private long customerId;
+
+    @Parameter(names = ArgumentNames.MANAGER_ID, required = true)
+    private long managerId;
+  }
+
+  public static void main(String[] args) throws IOException {
+    LinkManagerToClientParams params = new LinkManagerToClientParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+
+      params.managerId = Long.parseLong("INSERT_MANAGER_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new LinkManagerToClient().runExample(googleAdsClient, params.customerId, params.managerId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /** Runs the example. */
+  private void runExample(GoogleAdsClient googleAdsClient, long clientCustomerId, long managerId) {
+    // This example assumes that the same credentials will work for both customers, but that may not
+    // be the case. If you need to use different credentials for each customer, then you may either
+    // update the client configuration or instantiate two clients, one for each set of credentials.
+    // Always make sure to update the configuration before fetching any services you need to use.
```

Comment:
Should this say "Always make sure you use a GoogleAdsClient with the proper credentials to fetch any services you need to use."